### DISCUSSION TITLE
亞馬遜 Alexa 讓小孩自製睡前故事!🛏️

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#895] [General]
新聞大綱:
新的 Alexa 說故事功能使用對話式和生成式 AI 來創作故事情節、圖像和配樂。孩子只要說「Alexa，編一個故事」，接著就可以用口頭的方式選擇故事的場景主題、人物等，做好設定後，Alexa 就會創作出一個獨特的動畫故事。 

心得:
我覺得這個Alexa不僅能讓兒童在想聽故事的時候發揮想像力，也能讓家長不用絞盡腦汁的拼命從網路上找更多兒童故事了。
AI製作出來的東西盡管有很多爭議，但是我認為它也讓人類看到了更多的可能性。
![image](https://user-images.githubusercontent.com/90883526/207208254-6b87981c-7127-4e03-bb88-840b8fc487cd.png)

**Comment by dabbie109303501** [comment]
新聞連結?
但ai編出來的故事是否有邏輯性跟是否適合小孩子聽還蠻值得深入探討的

**Comment by HuangHsinTzu** [comment]
是不是忘了放連結了?
我覺說故事是一個培養親子關係很棒的方式
雖然AIexa的這項新功能可以減少家長們的負擔(但對於有些家長來說可能不是負擔)
對於工作較忙碌的家長來說是是一個很好的幫手
但Alexa還是不能代替家長說故事阿
它沒有愛的成分，也沒辦法讓小孩們感受到溫暖

**Comment by yuxinlin93** [comment]
但在睡前讓小孩動腦編故事，應該會讓小孩更難入睡吧，畢竟都讓腦袋運作起來了

**Comment by Kotori0501** [comment]
雖然可以利用關鍵字生成一個故事
但總覺得似乎少了一個感情

**Comment by Evan0979** [comment]
會不會邊出一些兒童不宜的故事XD
我想到最近很夯的AI繪圖哈哈哈

**Comment by peggy0329** [comment]
之前有看到底下這篇文章讓我不經懷疑 Alexa 會不會變成殺人武器，也太可怕...
亞馬遜Alexa竟然向一名10歲女孩「發出挑戰」，要她拿銅板去接觸插入插座但露出一半的插頭。
https://www.bbc.com/zhongwen/trad/science-59819735

我認為現在語音助理真的需要持續的改善，不知道透過它來說故事，會不會有不合適的內容出現
而且這樣少了一個跟爸媽互動的方式，感覺生活會變得好冰冷

**Comment by ruthchen1020** [comment]
可能不能睡前想故事吧，可以清醒時玩玩具的時候紀錄小孩對故事或者幻想的喜好，去訂製一個故事，這樣會不會更加有AI客製化的感覺。畢竟yt推給我的影片我都不一定喜歡了，AI講得我可能也不喜歡。

**Comment by Zoe06060606** [comment]
雖然感覺能用關鍵字產生故事但好像會缺乏羅輯性

**Comment by CPCpyt** [comment]
我覺得這是能夠激發孩子想像力的技術，不過我想在實行上還會有更多的阻礙。
